functions/utils/types.ts

```diff
@@ -114,3 +114,27 @@ export type FriendInfo = Record<
     email: string;
   }
 >;
+
+export type ScheduleDeletionRequest = {
+  /**
+   * token of account that requested the schedule deletion
+   */
+  IDToken: string | void;
+  /**
+   * ID of the INVITEE if the deletion requester is the INVITER
+   * ID of the INVITER if the deletion requester is the INVITEE
+   */
+  otherUserId: string;
+  /**
+   * term that schedule version(s) belong to
+   */
+  term: string;
+  /**
+   * shared schedule version(s) for deletion
+   */
+  versions: string[] | string;
```

Comment:
this is kind of unusual, let's stick to an array. If the client wants to delete a single version they can pass a single version in an array.